src/view/placeholder/AnimatedPlaceholder.jsx

```diff
@@ -0,0 +1,47 @@
+import styled, { css, keyframes } from 'styled-components';
+import { css as transitionStyles } from '../animation';
+
+const animateIn = ({ height, width }) => keyframes`
+  from {
+    height: 0;
+    width: 0;
+  }
+  to {
+    height: ${height}px;
+    width: ${width}px;
+  }
+`;
+
+const animateOut = ({ height, width }) => keyframes`
```

Comment:
We will need to use a different approach for this. We might need to create a style tag with the (prefixed) keyframe animation inside and append it / remove it from the head as needed.

Reply:
I think we can get away with just adding a `transition: height` on the element rather than keyframes